content/post/culture_of_partnership.md

```diff
@@ -0,0 +1,74 @@
+
+* Give you and your team additional insight into underlying business complexities
+* Provide you a _source_ of new data that you can feed into the [data quality flywheel](https://www.locallyoptimistic.com/post/data-dies-in-darkness/)
+* Stimulate questions for interesting research or product opportunities
+* Evangelize for the value that your team provides (since most of your work doesn't directly impact the bottom line)
+
+While it may be obvious that it's important to have a culture of partnership, it's devilishly un-obvious how to go about _building_ a culture of partnership either when building a team from scratch or when trying to change the culture of an existing org -- in this post I'll talk about how I think about building a strong culture of partnership that can help an analytics team identify the most important opportunities in the business and work effectively with key stakeholders and partner teams to drive value.
+
+I'll break this post into N parts:
+* Team values
+* Field research 
+* Building relationships
+
+## Team values
+
+When I first started building the analytics team at [Harry's](www.harrys.com), one of the first things I did was I sat down and wrote out what I wanted the core values of my team to be -- the first two things on the list were that an analytics team should be _humble_ and _helpful_.
+
+These two values are critical to building an analytics team that embodies [<- XX FIND BETTER WORD] a culture of partnership (and, in turn, building a high-functioning analytics team). Humility is how analytics practitioners should engage both with potential partners but also with their own work -- hubris is the enemy of good research. Helpfullness is the best tool for helping the analytics team build effective partnerships with other teams that might not always understand what the analytics team _actually does_.
+
+### Humility
+Analytics teams work best when they 
+
+1. have the support of the rest of the organization behind them and 
+2. work to truly understand the use-cases, problems, and expertises in the rest of the business
+
+One of the joys of working in analytics is that you get to work on a variety of different problem domains -- moving from marketing, to product development, to software engineering. However, that often means that we are often dropping into a new problem space without complete knowledge of the domain or of the history of the team we're working with -- humility helps analytics team approach knew problems knowing that they are not the experts. 
+
+When an analytics team is humble, it makes the partners feel like the experts in their domain (which they are!) and it can help stem some of the turf-battles that can occur when an analytics team drops into a problem area with guns blazing ready to shake things up.
```

Comment:
Fully agreed. I wonder if you want to mention the tension between being humble/listening and proactively providing advice/guidance. Some partners will need a nudge whether they know it or not. How to do that in a humble way could be helpful.